fulp_modules/features/antagonists/bloodsuckers/code/powers/_powers.dm

```diff
@@ -28,15 +29,9 @@
 	/// Who can purchase the Power
 	var/purchase_flags = NONE // BLOODSUCKER_CAN_BUY|BLOODSUCKER_DEFAULT_POWER|TREMERE_CAN_BUY|VASSAL_CAN_BUY
 
-	// COOLDOWNS //
-	///Timer between Power uses.
-	COOLDOWN_DECLARE(bloodsucker_power_cooldown)
-
 	// VARS //
-	/// If the Power is currently active.
+	/// If the Power is currently active, differs from action cooldown because of how powers are handled.
 	var/active = FALSE
```

Comment:
following the comment above i can change this to is_action_active but it would look kinda of nasty, atleast for me